src/Php56/bootstrap.php

```diff
@@ -21,4 +21,16 @@ function hash_equals($knownString, $userInput) { return p\Php56::hash_equals($kn
 
         function ldap_escape($subject, $ignore = '', $flags = 0) { return p\Php56::ldap_escape($subject, $ignore, $flags); }
     }
+
+    // Missing functions in PHP 5.5.9 - affects 32 bit builds of Ubuntu 14.04LTS
+    // See https://bugs.launchpad.net/ubuntu/+source/php5/+bug/1315888
+    if (!function_exists('gzopen') && function_exists('gzopen64')) {
+        function gzopen($filename , $mode = 'r', $use_include_path = 0 ) { return gzopen64($filename, $mode, $use_include_path); }
```

Comment:
$mode doesn't default to 'r' per the doc (http://php.net/gzopen)
trailing space before `)`